tests/snapshots.rs

```diff
@@ -292,5 +297,5 @@ fn convert_glsl_quad() {
     let mut entry_points = naga::FastHashMap::default();
     entry_points.insert("vert_main".to_string(), naga::ShaderStage::Vertex);
     entry_points.insert("frag_main".to_string(), naga::ShaderStage::Fragment);
-    convert_glsl("quad-glsl", entry_points);
+    convert_glsl("quad-glsl", entry_points, Targets::SPIRV);
```

Comment:
You may wanna add `Targets::IR` to match what it was doing before.

Reply:
I was trying to reach you on matrix to ask about this :) Anyway, brought IR back here.